akka-docs/rst/scala/cluster-sharding.rst

```diff
@@ -262,8 +262,7 @@ Note that stopped entities will be started again when a new message is targeted
 Graceful Shutdown
 -----------------
 
-You can send the message ``ClusterSharding.GracefulShutdown`` message (``ClusterSharding.gracefulShutdownInstance
-in Java) to the ``ShardRegion`` actor to handoff all shards that are hosted by that ``ShardRegion`` and then the
```

Comment:
I think you just need change this(``ClusterSharding.gracefulShutdownInstance
``-in Java)